### ISSUE TITLE
Idea] Private instant messaging

### ISSUE THREAD

**Issue by sbeeee** [HelloZeroNet/ZeroNet#509]
Hi,

Is it possible to develop a private instant messaging ( like Tox ) ?

**Comment by TheNain38** [comment]
Why not directly use Tox? ...

**Comment by sbeeee** [comment]
Yes , and finally Tox has the Distributed Hash Table , so it is more interesting.

**Comment by deavmi** [comment]
ZeroMail? It's pretty fast.

**Comment by deavmi** [comment]
I think if we were to make ZeroChat it would only be as slow as ZeroMail.

**Comment by border0464111** [comment]
it's possible to have a zeronet site filling this purpus , but unfortunately it won't be done inside this repository as it's zeronet core repository. 

Someone will have to an initiative for a zeronet site.

**Comment by AnthyG** [comment]
[ThunderWave](https://github.com/AnthyG/ThunderWave) has public chat (based on ZeroChat), and private chat (based on ZeroMail, but more secure), and I'll be adding group chat ASAP!
